openflow13/match.go

```diff
@@ -141,6 +146,7 @@ func (m *MatchField) MarshalBinary() (data []byte, err error) {
 
 func (m *MatchField) UnmarshalBinary(data []byte) error {
 	var n uint16 = 0
+	var err error = nil
```

Comment:
Ah, I see that you copied the existing code... please do not assume that existing code is correct or following Go best practices because in many cases it is not 😄 

You could also delete the `= 0` here.  0 is the default value of all integer types in Go